sdk/cognitiveservices/azure-cognitiveservices-personalizer/README.rst

```diff
@@ -3,30 +3,19 @@ Microsoft Azure SDK for Python
 
 This is the Microsoft Azure Personalizer Client Library.
 
-This package has been tested with Python 2.7, 3.4, 3.5, 3.6 and 3.7.
+This package has been tested with Python 2.7, 3.5, 3.6 and 3.7.
```

Comment:
Seems personalizer needs to have "auto_update=false" like most of data-plane. I suggest you create a commit with that, and you "rebase --interactive" to remove the "personalizer" commit, and then you "push --force"

Reply:
Or just remove the commit :)